sql/core/src/main/scala/org/apache/spark/sql/execution/joins/HashJoin.scala

```diff
@@ -49,9 +49,9 @@ trait HashJoin {
 
     s"""
        |(${ExplainUtils.getOpId(this)}) $nodeName ${ExplainUtils.getCodegenId(this)}
-       |Left keys: ${leftKeys}
-       |Right keys: ${rightKeys}
-       |Join condition: ${joinCondStr}
+       |Left keys      : ${leftKeys}
+       |Right keys     : ${rightKeys}
+       |Join condition : ${joinCondStr}
```

Comment:
@srowen Here we need this line up for better visualization?

Reply:
I don't think the same logic applies, no. I don't think a change is necessary.

Reply:
Got it. Thanks @srowen